### PR TITLE
feat: add some german translations

### PR DESCRIPTION
**Pull request by henk84**

### CODE REVIEW

File: language/anope.de_DE.po [hunk 1]
```diff
@@ -1924,6 +1907,9 @@ msgstr "Zugriffs-Liste von %s:"
 msgid "Access to this command requires the permission %s to be present in your opertype."
 msgstr "Um Zugriff auf diesen Befehl zu haben benötigst Du die Berechtigung %s in Deinem Oper Type."
 
+msgid "Account"
+msgstr ""
```
Comment (SadieCat) [inline language/anope.de_DE.po:1911]: Translation missing here.

File: language/anope.de_DE.po [hunk 2]
```diff
@@ -8262,8 +8321,9 @@ msgid ""
 "access entries on the channel."
 msgstr ""
 
+#, fuzzy
```
Comment (SadieCat) [inline language/anope.de_DE.po:8324]: There was a minor English change here to fix a typo. If this isn't fuzzy then you can remove this line.

File: language/anope.de_DE.po [hunk 3]
```diff
@@ -9611,3 +9661,6 @@ msgstr "{Nickname | Kanal}"
 
 msgid "{nick | channel} memo-text"
 msgstr "{Nickname | Kanal} Memo-Text"
+
+#~ msgid "Password authentication required for that command."
+#~ msgstr "Password Authentisierung ist notwendig für diesen Befehl."
```
Comment (SadieCat) [inline language/anope.de_DE.po:9666]: As these lines are obsolete (replaced with *You must be logged into an account to use that command.*) you can remove this.

File: language/anope.de_DE.po (hunk 1 above)
Comment (henk84) [reply language/anope.de_DE.po:1911]: FWIW: update.sh added these lines, AFAICT

Comment (SadieCat) [reply language/anope.de_DE.po:1911]: Yeah that's fine. It's a new translation string added since the translation file was last updated.

**Pull request merged**